src/utils/url.ts

```diff
@@ -205,6 +205,7 @@ export const externalUrlAdditionalProps = {
   target: "_blank",
   rel: "noopener noreferrer",
 };
+Object.freeze(externalUrlAdditionalProps);
```

Comment:
Can we just move `Object.freeze` inside the function and it's return?

Reply:
Some times TS might complain the interface is `Readonly<T>` instead of `T` even when just reading, so as a workaround, I've started adding the freeze after the `const` has inferred the correct type.